File: Domain/Entities/Product.cs
namespace Domain.Entities
{
    public class Product : BaseEntity
    {
        public string Model { get; set; }
        public string ProductName { get; set; }
        public string ImageUrl { get; set; }
        public decimal Price { get; set; }
        public int TopologyId { get; set; }
        public Topology Topology {get; set; }
        public int FunctionId { get; set; }
        public Function Function { get; set; }
        public string Dimensions { get; set; }
        public string Description { get; set; }
    }
}

File: client/src/app/models/product.ts
import { Pipe, PipeTransform } from "@angular/core"

export interface IProduct {
  id: number
  model: string
  productName: string
  imageUrl: string
  price: number
  topology: string
  function: string
  dimensions: string
  description: string
}

@Pipe({
  name: "perDay"
})

export class PerDayPipe implements PipeTransform {
  transform(value: any) {
    return value + " / night"
  }
}

